examples/RFC2616.hs

```diff
@@ -1,15 +1,11 @@
 {-# LANGUAGE BangPatterns #-}
-module Main (main) where
+
+module RFC2616 where
```

Comment:
This changes the name from `Main` to `RFC2616`, so one loses the ability to compile this to an executable. Perhaps it would be worth factoring out the `main` method in this file as a separate `executable` component in `trifecta-examples.cabal`?